src/table/test/headerVisible.test.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, fixture } from "@open-wc/testing";
+
+import { Dropdown } from "../../dropdown";
+import { Table } from "../index";
+
+const renderGender = (cellData: any) => {
+  const dropdown = new Dropdown({
+    id: "dropdown-gender",
+    items: [
+      {
+        label: "Male",
+        value: "male",
+      },
+      {
+        label: "Female",
+        value: "female",
+      },
+    ],
+    value: cellData,
+  });
+  return dropdown;
+};
+const columns = [
+  { field: "name", title: "Name" },
+  { field: "age", title: "Age" },
+  { field: "gender", title: "Gender", render: renderGender },
+];
+const data = [
+  {
+    name: "David",
+    age: 1,
+    gender: "male",
+    address: { city: "hcm", country: "vn" },
+    hobby: ["game", "flower"],
+  },
+  {
+    name: "Mary",
+    age: 2,
+    gender: "female",
+    address: { city: "tokyo", country: "ja" },
+    hobby: ["music"],
+  },
+  {
+    name: "Linda",
+    age: 3,
+    gender: "female",
+    address: { city: "tokyo", country: "ja" },
+    hobby: ["flower"],
+  },
+];
+describe("Table", () => {
+  describe("headerVisible", () => {
+    it("should be displayed when not assigned in constructor", async () => {
+      const container = new Table({ columns: [...columns], data: [...data] });
+      const el = await fixture(container);
+      const header = el.querySelector(
+        ".kuc-table__table__header"
+      ) as HTMLDivElement;
+      expect(header.hasAttribute("hidden")).to.equal(false);
+    });
+    it("should be displayed none when assigned false in constructor", async () => {
+      const container = new Table({
+        columns: [...columns],
+        data: [...data],
+        headerVisible: false,
+      });
+      const el = await fixture(container);
+      const header = el.querySelector(
```

Comment:
Please change "header" variable to "tableHeader".
I get confused between HTML tag header "<header>"